src/main/kotlin/com/github/warningimhack3r/intellijshadcnplugin/ui/ISPPanelPopulator.kt

```diff
@@ -23,7 +23,8 @@ class ISPPanelPopulator(private val project: Project) {
         CoroutineScope(SupervisorJob() + Dispatchers.Default).async {
             return@async Pair(
                 SourceScanner.findShadcnImplementation(project),
-                FileManager.getInstance(project).getVirtualFilesByName("package.json").size
+                FileManager.getInstance(project).getVirtualFilesByName("package.json")
+                    .filter { !it.parent.name.startsWith('.') }.size
```

Comment:
You can add this directly inside the `getVirtualFilesByName` function instead

Reply:
I had an issue where it detected the file in `.next`. This fixed it